docs/iris/example_code/General/polar_stereo.py

```diff
@@ -15,9 +15,17 @@
 
 
 def main():
-    # Enable a future option, to ensure that the grib load works the same way
-    # as in future Iris versions.
-    iris.FUTURE.strict_grib_load = True
+    # Enable a future option, to ensure that the grib load uses the mechanism
+    # intended for future Iris versions.
+    iris.FUTURE.external_grib_support = True
+
+    # What if anything is needed here for the first version of iris_grib ?
+    # - we don't actually know yet ...
+    # What we might need in future...
+    #    import iris_grib
+    #    iris_grib.FUTURE.strict_grib_load = True
+    # FOR NOW: a nasty kluge, as setting this is now *itself* deprecated.
+    iris.FUTURE.__dict__['strict_grib_load'] = True
```

Comment:
Note: when iris_grib is available, this becomes "**iris_grib**.FUTURE.\_\_dict__\['strict_grib_load'] = True"